src/LibHdb++MySQL.cpp

```diff
@@ -689,9 +730,27 @@ int HdbPPMySQL::insert_error(string error_desc, int &ERR_ID)
 		stringstream tmp;
 		tmp << "ERROR in query=" << query_str.str() << ", err=" << mysql_stmt_error(pstmt);
 		cout << __func__<< ": " << tmp.str() << endl;
+		if(!cached)
```

Comment:
Could it be factorised with the code line 734 ? might not be worse it, but it's the exact same code…

Reply:
Yes, going to do it.